lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { Fragment } from "react";
+import PropTypes from "prop-types";
+import Particles from "../../../../Agency/Particle";
+import BannerWrapper, {
+  DiscountLabel,
+  BannerMask,
+  Hspace,
+} from "./bannerSection.style";
+import { Box, Text, Heading, Button, Container } from "../../../../../atoms";
+import { FeatureBlock } from "../../../../../molecules";
+
+const BannerSection = ({
+  row,
+  col,
+  title,
+  btnStyle,
+  description,
+  discountText,
+  discountAmount,
+  outlineBtnStyle,
+  model,
+  isAR,
+}) => {
+  let data = model.data.reduce(function (map, obj) {
```

Comment:
```suggestion
  const data = model.data.reduce(function (map, obj) {
```
Let's convert it into const whenever we update a section, or whenever its suitable